modules/system/classes/PluginManager.php

```diff
@@ -253,6 +253,14 @@ public function registerPlugin($plugin, $pluginId = null)
             require $initFile;
         }
 
+        /*
+         * Add helpers
+         */
+        $helpersFile = $pluginPath . '/helpers.php';
+        if (File::exists($helpersFile)) {
+            require $helpersFile;
```

Comment:
Have you tested this? I wouldn't think that it would include the helpers.php file globally since this is happening within the scope of the registerPlugin() method.

Reply:
Yes, I tested this before posting. In this case, the logic works by analogy with connecting routes, the file is require into the system during the registration of the plug-in if it exists.

Reply:
routes.php and init.php are different in that they are including code that should be executed immediately, not defining global functions. I will have to test this myself before I can merge.

Reply:
During testing, I used a tinker, and also implemented a dependent plugin.

> routes.php and init.php are different in that they are including code that should be executed immediately, not defining global functions. I will have to test this myself before I can merge.

Oh sure. I will be glad to your check